ImperatorToCK3/Data_Files/configurables/province_mappings.txt

```diff
@@ -1,5 +1,259 @@
 0.0.0.0 = {
 	link = { comment = "=== NEW MAPPINGS BY RADU ===" }
+	link = { comment = "GERMANY, AUSTRIA, BOHEMIA BY RADU" }
+	link = { imp = 3667 ck3 = 3137 } # Aguntum -> OBERVELLACH
+	link = { imp = 3664 ck3 = 2952 ck3 = 3134 ck3 = 2951 } # Sebatum -> BRUNECK, LIENZ, MATREI
+	link = { imp = 3752 ck3 = 2943 } # Ambrae -> FREISING
+	link = { imp = 3751 ck3 = 2945 } # Abodiacum -> MUNCHEN
+	link = { imp = 3641 ck3 = 2947 } # Arces Covelicae -> WOLFRATSHAUSEN
+	link = { imp = 3642 ck3 = 2984 } # Isinisca -> MARQUARDSTEIN
+	link = { imp = 3753 ck3 = 2958 } # Bratananium -> ERDING
+	link = { imp = 3764 imp = 3754 ck3 = 2973 ck3 = 2981 ck3 = 2961 } # MANY-TO-MANY: Innaba, Ad Pontes Tessenios -> BRAUNAU, BURGHAUSEN, DORNBERG

```

Comment:
Many to many, intended?

Reply:
No, will correct. Also do you have any objections to the following suggestions from Iohannes from Discord?

Brigantium should map to Bregenz somehow, due to name continuity
Aquae Mattiacorum should map to Wiesbaden first, Katzenelnbogen and Limburg second (same reason)

Reply:
No objections, cities should always be mapped to the same cities in CK3 if they exist.

Reply:
Resolved Many to Many, implemented requested changes.